src/main/resources/META-INF/rewrite/java-version-11.yml

```diff
@@ -196,4 +200,62 @@ recipeList:
       groupPattern: com.sun.xml.ws
       artifactPattern: "*"
       onlyIfVersionsMatch: true
-
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.InternalBindContextFactory
+displayName: Use `com.sun.xml.bind.v2.ContextFactory` instead of `com.sun.xml.internal.bind.v2.ContextFactory`
+description: Do not use the com.sun.xml.internal.bind.v2.ContextFactory class.
+tags:
+  - java11
+recipeList:
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.sun.xml.internal.bind.v2.ContextFactory
+      newFullyQualifiedTypeName: com.sun.xml.bind.v2.ContextFactory
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JREDoNotUseSunNetSslInternalWwwProtocolHttpsHandler
+displayName: Use `com.ibm.net.ssl.www2.protocol.https.Handler` instead of `com.sun.net.ssl.internal.www.protocol.https.Handler`
+description: Do not use the com.sun.net.ssl.internal.www.protocol.https.Handler class.
+tags:
+  - java11
+recipeList:
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.sun.net.ssl.internal.www.protocol.https.Handler
+      newFullyQualifiedTypeName: com.ibm.net.ssl.www2.protocol.https.Handler
+      ignoreDefinition: true
```

Comment:
I'm doubting if we should migrate folks over to the ibm classes; would those not be constrained to a particular JVM vendor or dependency?

Curious to hear your thoughts as I might lack some background.

Reply:
This is one of the recipes I mentioned would be IBM-specific. It's possible other solutions exist, but this is one we provided an alternative for. This recipe would really be aimed at our customers. If you want I can remove the recipe from the general Java 11 list.

Reply:
I think removing it from the java-version-11.yml would be best indeed; would it fit in better with rewrite-liberty perhaps? Or would you think a separate IBM, OpenJ9 or Semuru yml file would fit in best here?

Reply:
A separate ibm-java yml would be a better fit.

Reply:
I'll push that update soon. I hadn't gotten a chance to revisit this PR until today. I just noticed that the build is passing now. Thanks for helping me work out those issues.